services/diagnostic/api.go

```diff
@@ -0,0 +1,95 @@
+package diagnostic
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/influxdata/kapacitor/services/httpd"
+	"github.com/influxdata/kapacitor/uuid"
+)
+
+const (
+	sessionsPath = "/logs"
+)
+
+//type Diagnostic interface {
+//}
+
+type SessionService struct {
+	//diag   Diagnostic
+	routes []httpd.Route
+
+	SessionsStore SessionsStore
+	HTTPDService  interface {
+		AddRoutes([]httpd.Route) error
+	}
+}
+
+func NewSessionService() *SessionService {
+	return &SessionService{
+		SessionsStore: &sessionsStore{
+			sessions: make(map[uuid.UUID]*Session),
+		},
+	}
+}
+
+func (s *SessionService) Close() error {
+	return nil
```

Comment:
In close you should remove the routes from the HTTPD service.

Reply:
fixed